### ISSUE TITLE
get_ssurgo issues

### ISSUE THREAD

**Issue by dcurles12** [leombastos/2024_ppa#33]
I am having an issue with this code and have not been able to figure it out. If anyone can help that would be great!

<img width="866" alt="Screenshot 2024-02-15 at 11 09 41 PM" src="https://github.com/leombastos/2024_ppa/assets/155988196/1197d639-12f6-4589-91d9-805bc3b5fef9">

**Comment by KVUGA** [comment]
Yes, The way I see it, and I think have resolved it is as follows:
Here, you are using the exact same code that has been printed in class. The issue with it is that the geospatial data saved here is with respect to the field we have studied in the class. So, in the section of "ssurgo", we are unable to rewrite the new information of our latest fields.
The way it could be resolved is that renaming it to something new such as "ssurgonew" in the lines after 118.
This will help save the latest data in the "ssurgonew" as per your field. I hope it works.

**Comment by leombastos** [comment]
@dcurles12 what Kumar is saying could help.
I also see on your code that you are missing a parenthesis in row 119 after `as("Spatial"`.
It should be `as("Spatial")`.
